luigi_pipeline/seqr_loading.py

```diff
@@ -69,7 +69,7 @@ def run(self):
         # first validate paths
         for source_path in self.source_paths:
             check_if_path_exists(source_path, "source_path")
-        check_if_path_exists(self.reference_ht_path, "reference_ht_path")
+        if self.reference_ht_path: check_if_path_exists(self.reference_ht_path, "reference_ht_path")
```

Comment:
out of scope but tests now fail because we're running through this pathway now.  I think this is probably fine for now... (clinvar has the same logic and is also required and we're going to fail when we try to load the reference data anyways)... but should realistically we should perform this check if `datasetType === SNV or datasetType === MITO`.